tv11.c

```diff
@@ -333,6 +333,8 @@ usage(void)
 	fprintf(stderr, "\tserver: host running pdp-10\n");
 	fprintf(stderr, "\t-p: pdp-10 port; default 1110\n");
 	fprintf(stderr, "\t-l: tv11 listen port; default 11100\n");
+	fprintf(stderr, "\t-s: sleep (ms); detault=0\n");
```

Comment:
Typo; should be default.  Also in xgp11.c.

Reply:
this should be fixed now